client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -169,7 +169,7 @@ const SiteMigrationInstructions: Step = function ( { navigation, flow } ) {
 			<div className="site-migration-instructions__steps">
 				<Steps steps={ steps } />
 			</div>
-			<Provisioning status={ detailedStatus } />
+			<Provisioning status={ detailedStatus } preparationCompleted={ preparationCompleted } />
```

Comment:
Can we infer the completion using only the detailed status ?

Reply:
Yes, we can, since it is already inferred from the status of the last two actions.
Thank you for pointing it out.